amass/sources/archiveit_test.go

```diff
@@ -12,40 +10,47 @@ import (
 )
 
 func TestArchiveIt(t *testing.T) {
-	config := &core.Config{}
-	config.AddDomain("google.com")
-	buf := new(strings.Builder)
-	config.Log = log.New(buf, "", log.Lmicroseconds)
-
-	out := make(chan *core.Request)
-	bus := core.NewEventBus()
-	bus.Subscribe(core.NewNameTopic, func(req *core.Request) {
-		out <- req
-	})
-	defer bus.Stop()
-
-	srv := NewArchiveIt(config, bus)
-	srv.Start()
-	defer srv.Stop()
-
-	count := 0
-	expected := 10
-	done := time.After(time.Second * 30)
-
-loop:
-	for {
-		select {
-		case <-out:
-			count++
-			if count == expected {
-				return
+	if *networkTest {
```

Comment:
Lines like this one should probably be:

```suggestion
	if *networkTest == false {
	    return
	}
```